amd64/src/semantic.rs

```diff
@@ -747,7 +755,7 @@ pub fn call(a: Rvalue) -> Result<(Vec<Statement>, JumpSpec)> {
         call ?, new_rip:64;
     });*/
     let stmts = rreil!{
-        call ?, (a);
```

Comment:
An alternative way would be to insert a new mnemonic `__call_summary` after each call that contains the effects of the call before. So a functions trashing `ECX` and `RAX` would look like this:
```rust
call <blah>
  call ? uuid-of-blah
__call_summary
  mov ECX:32, ?
  mov RAX:64, ?
```
This has the advatage that we won't have "weird" instructions and can model functions that return values on the stack.